frontend/src/metabase/query_builder/actions/core/core.js

```diff
@@ -231,11 +231,6 @@ export const apiUpdateQuestion = (question, { rerunQuery = false } = {}) => {
     // (some of the old alerts might be removed during update)
     await dispatch(fetchAlertsForQuestion(updatedQuestion.id()));
 
-    // remove the databases in the store that are used to populate the QB databases list.
-    // This is done when saving a Card because the newly saved card will be eligible for use as a source query
-    // so we want the databases list to be re-fetched next time we hit "New Question" so it shows up
-    dispatch(setRequestUnloaded(["entities", "databases"]));
```

Comment:
This line doesn't work because it needs to be `["entities", "databases_list"]` to unload lists.